hpcflow/sdk/helper/helper.py

```diff
@@ -146,6 +197,19 @@ def restart_helper(
     start_helper(app, timeout, timeout_check_interval, watch_interval, logger=logger)
 
 
+def strip_helper_PID(app):
+    PID_file = get_PID_file_path(app)
+    if not PID_file.is_file():
+        print("Helper not running!")
+        return None
+    else:
+        with PID_file.open("rt") as fp:
+            pidlines = fp.readlines()
+            for i in range(4):
+                pidlines[i] = pidlines[i].strip("pid =tmeou-chknrvalw\n")
```

Comment:
Could you make this a bit more readable/explicit? You could split on the equals sign like this: `pidlines[i].split("=")[1].strip()` My concern with the current code is if we were to add a new line/variable, it's not instantly clear how to modify the code to support it.

Reply:
Resolved in [e19c5ed](https://github.com/hpcflow/hpcflow-new/pull/340/commits/e19c5edf33d6194fcb135e9f7c3f0098cc445ac9).
I changed the way I am reading in the variables, because with a dictionary the new method is simpler...